src/dlgTriggerEditor.cpp

```diff
@@ -407,9 +407,9 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
             tr(R"(Saves your entire profile (triggers, aliases, scripts, timers, buttons and keys, but not the map or script-specific settings); also "synchronizes" modules that are so marked.)"));
     connect(mProfileSaveAction, SIGNAL(triggered()), this, SLOT(slot_profileSaveAction()));
 
-    QAction* saveProfileAsAction = new QAction(QIcon(QStringLiteral(":/icons/utilities-file-archiver.png")), tr("Save Profile As"), this);
-    saveProfileAsAction->setEnabled(true);
-    connect(saveProfileAsAction, SIGNAL(triggered()), this, SLOT(slot_profileSaveAsAction()));
+    mProfileSaveAsAction = new QAction(QIcon(QStringLiteral(":/icons/utilities-file-archiver.png")), tr("Save Profile As"), this);
+    mProfileSaveAsAction->setEnabled(true);
```

Comment:
ℹ️ As it happens the default case **is** enabled so this is unnecessary, but no matter to this PR. 😉